### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by miozune**

### CODE REVIEW

File: src/main/java/pers/gwyog/gtneioreplugin/util/GT5CFGHelper.java [hunk 1]
```diff
@@ -34,25 +34,25 @@ public static String GT5CFGSmallOres(String Veinname) {
                 raw.add(p.getName() + "=" + p.getBoolean());
             }
         }
-        if (!raw.isEmpty()) for (int i = 0; i < raw.size(); i++) {
+        if (!raw.isEmpty()) for (String s : raw) {
```
Comment (wlhlm) [inline src/main/java/pers/gwyog/gtneioreplugin/util/GT5CFGHelper.java]: Can you wrap this in {}?

File: src/main/java/pers/gwyog/gtneioreplugin/util/GT5CFGHelper.java [hunk 2]
```diff
@@ -135,11 +135,11 @@ public static String GT5CFG(String Veinname) {
             } while (st != null);
             reader.close(); // not needed anymore
 
-            if (!raw.isEmpty()) for (int i = 0; i < raw.size(); i++) {
+            if (!raw.isEmpty()) for (String s : raw) {
```
Comment (wlhlm) [inline src/main/java/pers/gwyog/gtneioreplugin/util/GT5CFGHelper.java]: {}

File: src/main/java/pers/gwyog/gtneioreplugin/util/GT5CFGHelper.java [hunk 3]
```diff
@@ -135,11 +135,11 @@ public static String GT5CFG(String Veinname) {
...
                 // filter needed booleans from raw
                 /// FMLLog.info("raw contains"+raw.get(i));
                 for (int j = 0; j < DimensionHelper.DimName.length; j++)
```
Comment (wlhlm) [inline src/main/java/pers/gwyog/gtneioreplugin/util/GT5CFGHelper.java]: {}

File: src/main/java/pers/gwyog/gtneioreplugin/util/CSVMaker.java [hunk 4]
```diff
@@ -24,9 +24,7 @@ public CSVMaker() {}
     public static List<Oremix> Combsort(List<Oremix> liste) {
         try {
             List<Oremix> liste2 = new ArrayList<Oremix>(liste.size());
```
Comment (wlhlm) [inline src/main/java/pers/gwyog/gtneioreplugin/util/CSVMaker.java]: So, all of those variable names are in German 😄 Would you be willing to refactor this into English?

**Review by wlhlm — CHANGES_REQUESTED** [review]

File: src/main/java/pers/gwyog/gtneioreplugin/util/CSVMaker.java (hunk 4 above)
Comment (miozune) [reply src/main/java/pers/gwyog/gtneioreplugin/util/CSVMaker.java]: Didn't even know it's German 🤣

File: src/main/java/pers/gwyog/gtneioreplugin/util/CSVMaker.java [hunk 5]
```diff
@@ -21,31 +21,29 @@ public class CSVMaker implements Runnable {
...
-            }
+            List<Oremix> list2 = new ArrayList<>(list.size());
+            list2.addAll(list);
 
-            int schritt = liste2.size();
-            boolean vertauscht = false;
+            int schritt = list2.size();
+            boolean vertauscht;
             do {
                 vertauscht = false;
```
Comment (wlhlm) [inline src/main/java/pers/gwyog/gtneioreplugin/util/CSVMaker.java]: vertauscht -> swapped

File: src/main/java/pers/gwyog/gtneioreplugin/util/CSVMaker.java [hunk 6]
```diff
@@ -21,31 +21,29 @@ public class CSVMaker implements Runnable {
...
                 if (schritt > 1) {
```
Comment (wlhlm) [inline src/main/java/pers/gwyog/gtneioreplugin/util/CSVMaker.java]: schritt -> step

**Review by wlhlm — CHANGES_REQUESTED** [review]
> Didn't even know it's German 🤣

Let me help you with that then.

**Review by wlhlm — APPROVED** [review]

**Pull request merged**